openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -61,9 +61,9 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
                     ${HTML(course_home_message_fragment.body_html())}
                 % endif
 
-                % if welcome_message_fragment and UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id):
+                % if message_fragment and UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id):
```

Comment:
Sorry I didn't notice this earlier, but "message_fragment" is too generic a name when "course_home_message_fragment" is used just above.  
Options:
1. Rename to "welcome_or_update_message_fragment".  @andy-armstrong likes short names so won't like this, but I like long clear names.
2. Rename to another suggestion?
3. Add a comment.  The problem with this is comments can get out of sync, and do you add the comment next to every usage?